frontend/src/modules/root/components/WhatNewSection/index.tsx

```diff
@@ -0,0 +1,82 @@
+import Button from 'common/components/Button';
+import Card from 'common/components/Card';
+import Container from 'common/components/Container';
+import React from 'react';
+import { HiOutlineArrowRight } from 'react-icons/hi';
+import {
+  CardContainer,
+  PortletDiscription,
+  PortletTitle,
+  PortletTitleHeader,
+  ShowAllButton,
+  WhatNewStyle,
+} from './styled';
+
+const WhatNewSection: React.FC = () => {
+  const constants = [
```

Comment:
missing type for `constants`, it should have own model that will help a lot when u binding data to API